t/app/controller/reports.t

```diff
@@ -318,6 +319,40 @@ subtest "it allows body users to filter by subtypes" => sub {
     };
 };
 
+subtest "it allows cobrands to set default body user filters" => sub {
+    my $cobrand = Test::MockModule->new('FixMyStreet::Cobrand::Default');
+    $cobrand->mock('on_map_default_body_user_status', sub { ('confirmed', 'investigating') });
+
+    FixMyStreet::override_config {
+        MAPIT_URL => 'http://mapit.uk/'
+    }, sub {
+        my $body = FixMyStreet::App->model('DB::Body')->find( $body_edin_id );
+        my $user = $mech->log_in_ok( 'test@example.com' );
+        $user->update({ from_body => $body });
+
+        my ($investigating_problem) = $mech->create_problems_for_body(1, $body_edin_id, 'Investigating report');
+        my ($scheduled_problem) = $mech->create_problems_for_body(1, $body_edin_id, 'A Scheduled report');
+        my ($in_progress_problem) = $mech->create_problems_for_body(1, $body_edin_id, 'In progress report');
+
+        $investigating_problem->update({ state => 'investigating' });
+        $scheduled_problem->update({ state => 'action scheduled' });
+        $in_progress_problem->update({ state => 'in progress' });
+
+        $mech->get_ok('/reports/City+of+Edinburgh+Council');
+        $mech->content_contains('<option value="investigating" selected>Investigating</option>');
```

Comment:
Should this also check confirmed is selected?

Reply:
I think this whole test goes actually if `on_map_default_body_user_status` is being removed